easybuild/easyconfigs/p/protobuf/protobuf-3.6.1-foss-2018b.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'protobuf'
+version = '3.6.1'
+
+homepage = 'https://github.com/google/protobuf/'
+description = """Google Protocol Buffers"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['https://github.com/google/protobuf/releases/download/v%(version)s/']
+sources = ['protobuf-cpp-%(version)s.tar.gz']
```

Comment:
Why the change to protobuf-cpp?
Older protobuf versions (currently in develop) uses protobuf-3.x.y.tar.gz